stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -267,14 +278,14 @@ extension Tensor where Scalar : TensorFlowFloatingPoint {
   static func _vjpSubtract(
     lhs: Tensor, rhs: Scalar
   ) -> (Tensor, (Tensor) -> (Tensor, Scalar)) {
-    return (lhs - rhs, { v in (v, 0 - v.sum().scalarized()) })
```

Comment:
This is some legacy code introduced in the early days when a `where` clause on `@differentiable` was not supported. Now it is fixed for better.